### ISSUE TITLE
Data in the hover over on assignment bar elements in the list of assignments can't be navigated to using keyboard

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/my-learning-analytics#800]
## Expected behavior (A clear and concise description of what you expected to happen) : 

Users should be able to navigate to or access all content in the Assignment Planning view by using the keyboard.

## Describe the bug (Tell us what happens instead of the expected behavior) :

From the 10/28/19 accessibility review:

"Progress toward Final Grade table elements are not keyboard operable. ... Some users will not be able to access the metadata available on hover (name, due, grade, points possible, etc)."

For the new Assignment Planning, there is no mouse-over in the progress bar.  Instead, these metadata are available in the individual assignment bars next to each assignment in the list of assignments.  The functional need is the same. Tabbing should allow users to access the information that is in the mouse-over data.

## Additional context (Add any other context about the problem here) :

This may be addressed as part of work on issue #691 and PR #702. This issue may also be mitigated or resolved by implementing narrative alternatives to charts, such as that discussed in issue #749.

**Comment by jennlove-um** [comment]
Hold off on this one and revisit after the Assignment Planning redesign, since it may not be an issue after that.  Assignments & mouse-over won't exist in the new view.

**Comment by jennlove-um** [comment]
Test keyboard accessibility for Assignment Planning redesign.

**Comment by jennlove-um** [comment]
Testing indicates that while users can tab to the individual assignment bars, they cannot view the data that is available in the hover-over for the assignment.

**Comment by jennlove-um** [comment]
Testing passes: Users can tab to the information bar for individual assignments and see the pop up. Tabbing off the information bar successfully closes the pop up.
